src/models/flinkStatement.ts

```diff
@@ -167,19 +201,18 @@ export class FlinkStatement implements IResourceBase, IdItem, ISearchable, IEnvP
 
   /**
    * For statement results to be viewable, it must satisfy these conditions:
-   * 1. The statement must NOT be a background statement (an INSERT INTO statement)
-   * 2. The statement must have been created in the last 24 hours
+   * 1. The statement must have been created in the last 24 hours
    *    (which is the TTL for the statement result to be deleted)
-   * 3. The statement phase must be either RUNNING or COMPLETED.
+   * 2. The statement phase indicates viewable {@see VIEWABLE_PHASES}
```

Comment:
Slightly easier linking that supports clicking `viewable` to go straight to the `VIEWABLE_PHASES` definition:
```suggestion
   * 2. The statement phase indicates {@link VIEWABLE_PHASES viewable}
```
Before:
<img width="531" alt="image" src="https://github.com/user-attachments/assets/579624f2-fb14-4d6a-ab63-1dd623795de8" />

After:
<img width="471" alt="image" src="https://github.com/user-attachments/assets/ac71a35d-10e9-4471-97cb-0bf75c4f263b" />

Reply:
Thanks!